client/landing/gutenboarding/onboarding-block/acquire-intent/site-title.tsx

```diff
@@ -50,7 +50,7 @@ const SiteTitle: React.FunctionComponent< Props > = ( { onSubmit } ) => {
 					spellCheck={ false }
 					autoComplete="off"
 					autoCorrect="off"
-					data-hj-whitelist
+					data-hj-whitelist // @todo(saramarcondes) where is this used?
```

Comment:
This is to allow Hotjar.com show what user types into this field and unfortunately we cannot modify it.

https://help.hotjar.com/hc/en-us/articles/115015563287-How-to-Whitelist-Elements-and-Keystrokes

Reply:
Ah okay, thanks for letting me know. I’ll add a note in a comment!

Reply:
I reached out to HotJar yesterday and got a response from them that they're working on this internally. They're going to let me know once the data attribute has been updated to be able to be changed to something different.

Reply:
Excellent! Thanks for checking with them.

Reply:
Looks like the new attribute name is `data-hj-allow`.
https://help.hotjar.com/hc/en-us/articles/115015563287-How-to-Whitelist-Elements-and-Keystrokes

Updating eslint exception here: #46620